src/platform/packages/shared/kbn-esql-ast/src/definitions/utils/autocomplete/functions.ts

```diff
@@ -490,7 +494,7 @@ async function getListArgsSuggestions(
               userDefinedColumns: anyUserDefinedColumns,
             },
             { ignoreColumns: [firstArg.name, ...otherArgs.map(({ name }) => name)] },
-            hasMinimumLicenseRequired
+            callbacks
```

Comment:
It's not a real refactoring, but at least every time we make a change like this we don't change too many files